### DISCUSSION TITLE
v4.0.0

### DISCUSSION THREAD

**Discussion by Guillemsc** [Guillemsc/GTweens#5] [Announcements]
Changelog:
- Added `OnUpdate` method to the `GTween` class.
- Added `SetDelay `method to the `GTween `class.
- Updated versioning from minor to major.

<hr /><em>This discussion was created from the release <a href='https://github.com/Guillemsc/GTweens/releases/tag/v4.0.0'>v4.0.0</a>.</em>

**Comment by delustra** [comment]
Hey! Just wanted to stop by and say, that it's a great lib. I use on my project based on Stride Engine.
